### ISSUE TITLE
EEPROM support

### ISSUE THREAD

**Issue by agg23** [agg23/openfpga-arduboy#11]
Add support for EEPROM saving/loading

**Comment by Jepl4r** [comment]
Yes please. I have so many games, it would be amazing to have the possibility to save the progress.

**Comment by AnotherPocket** [comment]
Not only saving, but also for those games that use EEPROM to enable sound. Many of the games do not have sound because of this.

**Comment by AnotherPocket** [comment]
Hi. This was marked as completed on April 7th. Is there a binary? 

Thank you!

**Comment by ericlewis** [comment]
no, but we can get one setup!
